Formula/lgogdownloader.rb

```diff
@@ -0,0 +1,40 @@
+class Lgogdownloader < Formula
+  desc "Open source downloader for GOG.com games"
+  homepage "https://sites.google.com/site/gogdownloader"
+  url "https://sites.google.com/site/gogdownloader/lgogdownloader-3.2.tar.gz"
+  sha256 "9e60af50c18533884ca317778b291152d5f04d93dfd3dc2579f6f965538b2031"
+
+  depends_on "cmake" => :build
+  depends_on "help2man" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "htmlcxx"
+  depends_on "jsoncpp"
+  depends_on "liboauth"
+  depends_on "rhash"
+  depends_on "tinyxml2"
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    require "pty"
+
+    test_credentials = {
+      :username => "test@example.com\n",
+      :password => "secret\n",
+    }
+    ENV["XDG_CONFIG_HOME"] = pwd
+    reader, writer = PTY.spawn(
+      bin/"lgogdownloader", "--list", "--retries", "1"
+    )
+    writer.write(test_credentials.values.join(""))
+    writer.close
+    assert_equal("HTTP: Login failed",
```

Comment:
should all fit on one line and no parentheses needed

Reply:
Fixed, thanks.